app/src/main/java/com/atomicrobot/carbon/data/api/github/GitHubInteractor.kt

```diff
@@ -5,18 +5,23 @@ import androidx.annotation.VisibleForTesting
 import com.atomicrobot.carbon.Mockable
 import com.atomicrobot.carbon.R
 import com.atomicrobot.carbon.data.api.github.model.Commit
+import com.atomicrobot.carbon.data.api.github.model.DetailedCommit
 import retrofit2.Response
 import timber.log.Timber
 
 @Mockable
 class GitHubInteractor(
     private val context: Context,
-    private val api: GitHubApiService
+    private val api: GitHubApiService,
+    private val api2: DetailedGitHubApiService
```

Comment:
Optional: Generally here we would want to use `private val githubApiService: GitHubApiService` and `private val detailedGithubApiService: DetailedGitHubApiService`. Even though they are longer it keeps the dev from scrolling back up to find which one is which and helps to avoid programming mistakes by being more descriptive.